Azure-Samples/dotNET-Workers-with-gRPC-messaging-on-Azure-Container-Apps
# Deploy 1

## Purpose

* ...
Temporary fix for breaking upgrade to Azure CLI Docker container base Linux image

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ X] Other... Please describe:
Workaround for breaking change
```

## How to Test
*  Get the code
Deploy workflow should work without issues.

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
